ViewGeneratorCore/build/ViewGeneratorCore.targets

```diff
@@ -39,7 +39,7 @@
   </Target>
 
   <ItemGroup>
-    <VGC_Inputs Include="$(OverrideProjectDir)**\*.tsx" Exclude="$(OverrideProjectDir)node_modules\**\*.tsx" />
+    <VGC_Inputs Include="$(OverrideProjectDir)**\*.view.tsx" Exclude="$(OverrideProjectDir)node_modules\**\*.tsx" />
```

Comment:
view.tsx... is just a convention... cant be used here

Reply:
fixed, this pattern is now parsed from ts2lang.json config file